### ISSUE TITLE
Deprecate this application

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/next-tag-facets-api#39]
As discussed in https://github.com/Financial-Times/next-search-api/ apps should move away from this application to use the search API app instead.

- [ ] Lookup apps using this API (Graphite should have the data)
- [ ] Port this apps test suit over to the search API app
- [ ] Migrate apps using this API
  - [ ] TME search API (Also deprecated, might just require archiving)
  - [ ] next video editor app
  - [ ] stream page app
  - [ ] KMT tools and KMT API (don't know what these are)
  - [ ] Ads API

**Comment by matthew-andrews** [comment]
have you looked at the concept search api that UPP have made? (bernedette sent a note about tips/tricks from a tech point of view recently)  maybe it can replace this app as well?

**Comment by i-like-robots** [comment]
Yes, I've spoken to Bernadette about this (I think their work was somewhat based on ours!), and hopefully as we move over to a CAPI V2 metadata world we can lean on the UPP team more.
